portfolio/src/main/webapp/assets/js/main.js

```diff
@@ -70,19 +70,17 @@ jQuery(document).ready(function($) {
 
 async function getTestimonials() {
   const response = await fetch('/data');
-  const result = await response.json();
+  const arrTestimonials = await response.json();
+  console.log(arrTestimonials);
   
-  arrTestimonials = result["arrTestimonials"];
+//   arrTestimonials = result["arrTestimonials"];
```

Comment:
Should this be removed?

Reply:
Yes- I find that my process in past projects has always been to comment things out and then delete them later on because I often want to switch back. Trying to be more diligent about deleting these before submitting PR's but I keep forgetting- sorry about that.